neocore/Cryptography/ECCurve.py

```diff
@@ -570,11 +602,16 @@ def decompress(self, x, flag):
         return self.point(x, ysquare.sqrt(flag))
 
     def decode_from_reader(self, reader):
-
-        f = reader.ReadByte()
-
-        if f == 0:
+        """
+        Raises:
+            NotImplementedError: if the read int.from_bytes `f` == 4, 6, or 7
+            TypeError: if unexpected incoding is read
```

Comment:
```suggestion
            TypeError: if unexpected encoding is read
```
minor typo

Reply:
I wondered about this. I thought it was a term, I was unfamiliar with (there are several in the code).